cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/AppFabricServer.java

```diff
@@ -127,6 +129,7 @@ public AppFabricServer(CConfiguration cConf, SConfiguration sConf,
     this.bootstrapService = bootstrapService;
     this.systemAppManagementService = systemAppManagementService;
     this.capabilityManagementService = capabilityManagementService;
+    this.capabilityFeatureEnabled = cConf.getBoolean(Constants.Capability.FEATURE_ENABLED, false);
```

Comment:
Also, default value should be declared in cdap-default.xml instead of in code for easy finding.